src/index.js

```diff
@@ -3,6 +3,13 @@ import chProcess from 'child_process';
 import _ from 'lodash';
 import fs from 'fs';
 import chokidar from 'chokidar';
+import path from 'path';
+
+(function setPath() {
+  process.env.PATH = [
+    `${process.cwd()}/node_modules/.bin/`, process.env.PATH 
+  ].join(path.delimiter);
+})();
 
```

Comment:
Actually with putting process.env.PATH to `run` function we also have that problem, because it manipulates PATH globally, so as soon as `run` will be called  PATH will be modified. That could change behaviour of other child_process functions used by user.